.vscode/settings.json

```diff
@@ -18,5 +18,82 @@
         "./bazel-out/_coverage/_coverage_report.dat"
     ],
     "python.formatting.provider": "yapf",
-    "editor.formatOnSave": true
-}
+    "editor.formatOnSave": true,
+    "files.associations": {
```

Comment:
lol let's clean this part later. If we want to change it it should be a separate pr.